### PR TITLE
Add changes to selectfile.py to make it work on Windows

### PR DESCRIPTION
**Pull request by masaguaro**
A couple of lines changed in `selectfile.py`  to make it work on Windows

### CODE REVIEW

File: selectfile.py [hunk 1]
```diff
@@ -13,7 +13,8 @@ def _update_files(self):
         self.dirs = list()
         if(os.path.isdir(self.path)):
             for f in os.listdir(self.path):
-                ff = self.path + "/" + f
+                #ff = self.path + "/" + f
```
Comment (neomatrix369) [inline selectfile.py]: Remove this commented line, it's already saved in the VCS history

File: selectfile.py [hunk 2]
```diff
@@ -30,7 +31,8 @@ def on_click(b):
             if b.description == '..':
                 self.path = os.path.split(self.path)[0]
             else:
-                self.path = self.path + "/" + b.description
+                #self.path = self.path + "/" + b.description
```
Comment (neomatrix369) [inline selectfile.py]: remove commented line

**Review by neomatrix369 — CHANGES_REQUESTED** [review]
LGTM - apart from the two comments, its a good way to join path elements, so pls merge once you have resolved the two comments

**Comment by masaguaro** [comment]
Thank you for the comments @neomatrix369

**Pull request merged**